Tasks/Common/nuget-task-common/externals.json

```diff
@@ -0,0 +1,19 @@
+{
+    "archivePackages": [
+        {
+            "archiveName": "NuGet.exe",
+            "url": "https://vstsagenttools.blob.core.windows.net/tools/NuGet/3.3.0/nuget.zip",
```

Comment:
Is this used to download at runtime or build time?  That blob container store is our private store for the build process.  Just want to make sure it's not used at runtime because then we have a compat problem.  It will need to be there for the lifetime of the supported product (10 years) and we can't do that.  If you're pulling at runtime, you should look into public nuget acquisition locations.